service-front/module/Application/src/Controller/IndexController.php

```diff
@@ -55,8 +55,12 @@ public function startAction(): Response
         $type = $this->params()->fromQuery("personType");
 
         if (! $this->lpaFormHelper->lpaIdentitiesMatch($lpas, $type)) {
-            $personTypeDescription = $type === 'donor' ? "Donors" : " Certificate Providers";
-            throw new HttpException(400, "These LPAs are for different " . $personTypeDescription);
+            $personTypeDescription = [
+                'donor' => "Donors",
+                'certificateProvidor' => "Certificate Providers",
```

Comment:
Is typo on purpose?
(certificateProvidor -> certificateProvider)

Reply:
good spot!